### DISCUSSION TITLE
Unlock the app with fingerprint

### DISCUSSION THREAD

**Discussion by dmotte** [GitJournal/GitJournal#720] [Feature Requests]
## Environment

**App version:** 1.73.0+2015
**Device information:** (not relevant)
 - OS name and version: Android 9.0 Pie (not relevant)

## Description

Hi! First of all, congratulations for the great app! This is the note-taking app I've always dreamed of.

I would like to make a feature suggestion: it would be great if we can **lock the whole app** and unlock it only with the fingerprint sensor, for example like Aegis Authenticator does (see image below). Let me know if this is possible. No hurry, and thanks again for this beautiful app. Keep up the good work!

![image](https://user-images.githubusercontent.com/37443982/98939908-76434780-24ea-11eb-9c5d-1baef47f949d.png)

**Comment by vHanda** [comment]
Thanks! It's definitely possible. I just need to make it a priority. My task list for November is already quite full, so I don't want to add it, but let me see for December.

Related to #28

**Comment by roshanjhegde** [comment]
@vHanda great work with the app. Love it totally. Finger print 🔒 coming anytime soon?

**Comment by thoschworks** [comment]
You should add _Touch ID_ and _Face ID_ on the iOS/iPad versions too.

**Comment by vHanda** [comment]
> @vHanda great work with the app. Love it totally. Finger print 🔒 coming anytime soon?

This is the list of things I will try my best to deliver this month - https://github.com/GitJournal/GitJournal/milestone/4 . I do, however, tend to procrastinate and do other tasks. So maybe?

In general, if you can gather the votes, it'll go on next month's list.
